shell/platform/darwin/macos/framework/Source/FlutterDartProject.mm

```diff
@@ -89,4 +89,38 @@ - (NSString*)ICUDataPath {
   return arguments;
 }
 
++ (NSString*)flutterAssetsName:(NSBundle*)bundle {
```

Comment:
Private helpers in the macOS embedding need to be declared at the top of the file, and documented, as that's the prevailing style for this embedding.

This also needs to be named according to ObjC conventions, with the signature naming the parameter that follows it. `bundle` isn't an asset name, which is what the current naming suggests.

Reply:
This was marked as resolved without any change.

Reply:
Done.